src/Navigation/Xamarin.Forms/Navigation.Xamarin.Forms/Service/NavigationService.close.cs

```diff
@@ -89,6 +88,17 @@ public virtual async Task CloseAllAsync()
         }
 
 
+        public virtual async Task CloseModalAsync<TViewModel>()
+        {
+            if (Navigation == null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5598919)